tests/test-constructors.js

```diff
@@ -107,3 +107,27 @@ test('Make sure month May exists', function() {
   assert.equal(new Date(1598964466381).toDateString(), 'Tue Sept 01 2020');
   timezone_mock.unregister();
 });
+
+//////////////////////////////////////////////////////////////////////////
+test('option to use a fallback function when failing to parse (issue #24)', function() {
+  timezone_mock.unregister();
+  var _Date = Date;
+  timezone_mock.options({
+    fallbackFn: (date) => new _Date(date)
+  });
+  timezone_mock.register('UTC');
+  assert.equal(new Date('Fri, 26 Jul 2019 10:32:24 GMT').toDateString(), 'Fri Jul 26 2019');
+  timezone_mock.unregister();
+
+  // How can we reset options?
+  timezone_mock.options();
+  timezone_mock.register('UTC');
+  try {
+    assert.equal(new Date('Fri, 26 Jul 2019 10:32:24 GMT').toDateString(), 'Fri Jul 26 2019');
+    assert.error('We shouldn\'t have made it here!');
```

Comment:
If this assert get's hit the `catch (err)` below will catch it and ignore it, I think, maybe set a flag here and assert the flag isn't set (outside of the catch).

Otherwise, looks good, I can merge and push to NPM tomorrow when I have a chance.

Reply:
Yeah, thought about this afterwards. We could validate that we at least get the error we expect.